go/vt/vtgate/planbuilder/testdata/vschemas/schema.json

```diff
@@ -948,6 +948,61 @@
           ]
         }
       }
+    },
+    "sequential": {
```

Comment:
not sure, but I'm thinking we could use an existing keyspace instead of adding a new one. feel free to add a vindex or even a table if you need to.
I'm just thinking of not adding unnecessary complexity if we don't have to

Reply:
sure thing , incorporated with 5a7c8f1affcb237372da8fcd1a189778d0effef8 , using `user` keyspace - added extra vindex and table